web/src/hooks/UseExportLogs.ts

```diff
@@ -0,0 +1,62 @@
+import axios from 'src/libs/axios';
+import {ResourceError} from 'src/resources/ErrorHandling';
+
+export async function exportLogsForRepository(
+  org: string,
+  repo: string,
+  starttime: string,
+  endtime: string,
+  callback: string,
+) {
```

Comment:
nit:  Both of these function only differ in the axios `url` which relies on `repo` argument, rest of the code is same. I feel both of these can be combined into one function to avoid duplication, wdyt?

Reply:
agreed, I'll refactor this to be more concise